src/utils/error.js

```diff
@@ -0,0 +1,16 @@
+// @flow
```

Comment:
Added error.js based on warning.js which is just another wrapper fir logging errors. Invariant wouldn't replace errors as the same will throw error and break the code execution which will result into broken render

Reply:
Can we please use existing `warning`?
At this point formless fields are not an error per say. Its usage is expected, but yet not supported. I feel that a warning is a proper option here.

What do you think?

Reply:
Agree, we can deal it with warning for the time being until we have support for the formless field.

Should I remove the error.js file as well?

Reply:
Exactly. Once #216 is shipped, we can remove this warning.